.github/ISSUE_TEMPLATE/10-bug-report.yml

```diff
@@ -0,0 +1,61 @@
+name: Bug Report
+description: File a bug report related to running the package
+title: "[Bug] "
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+
+        Please, before submitting, make sure that:
+
+        - There is not an [existing issue](https://github.com/JuliaSmoothOptimizers/JSOSuite.jl/issues) with the same question
+        - You have read the [contributing guide](https://github.com/JuliaSmoothOptimizers/JSOSuite.jl/blob/main/docs/src/90-contributing.md/)
+        - You are following the [code of conduct](https://github.com/JuliaSmoothOptimizers/JSOSuite.jl/blob/main/CODE_OF_CONDUCT.md)
+
+        The form below should help you in filling out this issue.
```

Comment:
I think we should more directly ask for a Minimal reproducible example. From what I've seen 75% of the first comment from a question is "Do you have a MWE?"

Reply:
Will update the template. There is a question for reproduction steps, I will add more info there.